tutorials/magento-ubuntu/index.mdx

```diff
@@ -0,0 +1,378 @@
+---
+layout: docs
+title: Deploying Magento Open Source on Ubuntu 20.04
+description: Learn how to deploy the Magento e-commerce platform on a dedicated server using a LAMP stack.
+tags: compute dedicated server magento e-commerce onlineshop
+excerpt: |
+  Magento is one of the most popular e-commerce systems available, with more than 100.000 online stores based on the platform. It is written in PHP and uses the Zend Framework. Database-wise it uses the MySQL or MariaDB relational database system.
+image: /assets/scaleway_magento.png
+keywords: cloud tutorials, cloud documentations, cloud instance, cloud hosting, ubuntu server, debian server, linux server, IAAS, baremetal, e-commerce, magento, online shopping
+dates:
+  validation: 2021-09-21
+  posted: 2020-06-17
+---
+
+Magento is one of the most popular e-commerce systems available, with more than 100.000 online stores based on the platform. It is written in PHP and uses the Zend Framework. Database-wise it uses the MySQL or MariaDB relational database system.
+
+The software is currently owned by Adobe, and is available in a free, community-driven, open-source version which is distributed under the [Open Software License (OSL) v3.0](https://opensource.org/licenses/OSL-3.0), an Open Source Initiative approved license. A [commercial version](https://magento.com/products/magento-commerce) of the software is also available and provides advanced support and additional features.
+
+In this tutorial you will learn how to install the Open Source version of Magento on a [Scaleway Instance](https://www.scaleway.com/en/virtual-instances/) dedicated server running on Ubuntu Focal Fossa (20.04 LTS). To accomplish this mission we will deploy a LAMP stack (Ubuntu Linux, Apache2, MariaDB, and PHP) to install Magento 2 Open Source Edition on your Dedibox.
```

Comment:
c'est bizarre "to accomplish this mission"

Reply:
changed